environments/production/known_hosts

```diff
@@ -5,32 +5,28 @@
+172.24.32.67, ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC/+2z5IfNLeYd4wasIlNsIsldZc4Mb6tGx2w+1yHj26KJsJ/sXCzsnQdf74mYwfMKj3nFwvq8LWDOslx3/heX3+WrgDBtaG9FHvT6imkCiQq9/4RE8plACHGl4cFhwY/+wSwP7mcdLrUn5HDQ/8gclA+oVfenHjPKSntK9rHI6ri7I/LJAw3kCdK04ZslwgK0F0pzHbcrTmlyXDLrG3tzOmKOvaqOXCjKqYl3C4AlRZ+iAUAGQnYRtWJAecFb019EqxRgw1IHqlyYaZ1A80DigXcBqLlpdOe5/ZEEVu/K1zwOk2s5CaTvHxm18Djq3/jMgll/+GIiIIkF3Qcmo8t7t
 172.24.32.67,172.24.32.67 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBC3JqW+rY/FeSeZ+1W/73W0mpiYkJBARNZdNa4VCKmihpG48A2IFMCadz9Ps1cRpvYYAGbYPKbRlRdMKMcg/Lqk=
 172.24.32.67,172.24.32.67 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAILGG//Be0sl6fPn9l0RpFdUnOJeqdB6jH8pNJbcytngy
-172.24.32.67,172.24.32.67 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC/+2z5IfNLeYd4wasIlNsIsldZc4Mb6tGx2w+1yHj26KJsJ/sXCzsnQdf74mYwfMKj3nFwvq8LWDOslx3/heX3+WrgDBtaG9FHvT6imkCiQq9/4RE8plACHGl4cFhwY/+wSwP7mcdLrUn5HDQ/8gclA+oVfenHjPKSntK9rHI6ri7I/LJAw3kCdK04ZslwgK0F0pzHbcrTmlyXDLrG3tzOmKOvaqOXCjKqYl3C4AlRZ+iAUAGQnYRtWJAecFb019EqxRgw1IHqlyYaZ1A80DigXcBqLlpdOe5/ZEEVu/K1zwOk2s5CaTvHxm18Djq3/jMgll/+GIiIIkF3Qcmo8t7t
 control.internal-va.commcarehq.org,172.24.32.28 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFDfebBOcQHMYiQySNA+bGmksSRabcvVZPQgYu8MzdIAt7mbZ11JI3irhYC/oEm99r/m1lNb3O8KhvoTAeylCmM=
 control.internal-va.commcarehq.org,172.24.32.28 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDKnPdc+JjJI3pGghUzr/+cWLZtZtHOTZlSsAzq2eS1U
 control.internal-va.commcarehq.org,172.24.32.28 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDUhl6fhQMbqIcN1HoC0oARQX66DD54cUwGfR34B5hNuzMtVmfqD469wgPRWwPcWJFKBy4aB9XntwiJDSpU3fqOC9LYoT7yh8sR57LNnpfMPYeMKviJhwZauQc9zkoj7q3FoaZ65kQZpMgMLPXjWP9Et9suu4YdhS/aQlfcgixzG8OXuwKuDfzIGl8ceUQQjJsZxKm+7jA7ElnXbt/TAioUMVXinEasbnbn11AhxL9zOJJAIxQ1NsYOf1w3bn4hxnGT4wtwxqdAbL3o4TSPiS5MKyOPr7vXzdXgvql1mGNFT22O/4hL8wehrwJKLy6BYfrPblapCK+/+0rK7dRXhVxB
 hqcelery0.internal-va.commcarehq.org,172.24.32.13 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBLtaIJBzAQz8VQGPXpLlS+8Z+JadUvaEklgx6fOzZPg0HqOgF9TMXFi7dgJEIkbr/n3vYFfHg67Yaa1moNGpWt8=
 hqcelery0.internal-va.commcarehq.org,172.24.32.13 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAICK/myLm0lapD3ufeTR4fRy2Ssw6yYe+J4TjhNvbg2AP
 hqcelery0.internal-va.commcarehq.org,172.24.32.13 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDNwysSUb2RlX8/C8dx0HVwZou1KTkvF+aDAAgbjGIUEVsDKvnfY+6S9uxOQ3tXPNJYWYMl1608vRkVdf8JmOEOiTuFS+YaRJhln4IfxMhlBMZqjvmwhii6Nnskm2TCFZSG77+kV7PJqKMuP7tkHZZbgE1qKFMCASXUdRhmNWfc001p3UWBMclKzFW0zx8EH/K4RYznd/I6zx7Bn04pAqvZOOfTRHlfgkSO7zh848da6AJqIHur4ogMRnwC+ciUZrDChIIIwoQkPe12/f7aGLUlc1o9FsUSS6wvfoSVsU24yzIjE0Wj4RtUEbs9sDMknbOJC0TOH7FLyYnzU2dENNVN
 hqcelery1.internal-va.commcarehq.org,172.24.32.14 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBM09leVaWCkVHKuhcUxbCRSYBdUOEnQMV2dqXuM7of34EdIgwyHdM41epuCtRQ0JqpCA5pdf43yAgF8wUojKdjA=
-hqcelery1.internal-va.commcarehq.org,172.24.32.14 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAINKX5WIFLCIepkO5gYDE70v0OnkKHa/o+bFqh3lk2KZh
```

Comment:
i would expect only new entry for the db machine in this file and not others.
@snopoke saw something similar on Danny's PR, do you know why this is and if its okay?

Reply:
I've noticed differences if I run the command locally or on the control machine